GeneralsMD/Code/GameEngine/Source/Common/System/GameMemory.cpp

```diff
@@ -881,16 +881,16 @@ void MemoryPoolSingleBlock::initBlock(Int logicalSize, MemoryPoolBlob *owningBlo
 	m_checkpointInfo = NULL;
 #endif
 
-#ifdef MEMORYPOOL_BOUNDINGWALL
-	m_wallPattern = theBoundingWallPattern++;
-	debugFillInWalls();
```

Comment:
This called into `debugVerifyBlock` which tested `m_nextBlock` and `m_prevBlock`, but these would still be c3c3c3c3 (fill value) at this time.